license

```diff
@@ -1,9 +1,343 @@
-MIT License
+Node.js wrapper is implemented by Imagemin team and distributed under the MIT License. The Gifsicle code is Copyright (C) 1997-2021 Eddie Kohler and distributed under the GNU General Public License, Version 2 (and only Version 2) as below.
```

Comment:
Please note that this information is incomplete. See https://github.com/kohler/gifsicle#copyrightlicense

I would also challenge that you can build an MIT wrapper around a GPL component. If that would work I would write MIT wrappers for everything. A proper fix would be to license the project under GPL-2.0-only.